model/config.go

```diff
@@ -707,6 +709,14 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 			s.EnableLatex = NewBool(false)
 		}
 	}
+
+	if s.EnableLocalMode == nil {
+		s.EnableLocalMode = NewBool(false)
+	}
+
+	if s.LocalModeSocketLocation == nil {
+		s.LocalModeSocketLocation = NewString("/var/tmp/mattermost_local.socket")
```

Comment:
This can be a `const` at the beginning of the file and we can import it in `mmctl` for the default path

Reply:
Yes, definitely!